api/lib/mno_enterprise/concerns/controllers/jpi/v1/organizations_controller.rb

```diff
@@ -168,6 +168,17 @@ def remove_member
     render 'members'
   end
 
+
+    # POST /mnoe/jpi/v1/organizations/:id/freeze
+    def halt
+      # Check the password provided
+      return head :forbidden unless current_user.class.authenticate_user({email: current_user.email, password: params[:password]})
+      del_req = MnoEnterprise::DeletionRequest.create(deletable_type: "Organization", deletable_id: params[:id], reason: params[:reason])
```

Comment:
@x4d3 could you comment on what the best syntax to have here regarding associations?

Reply:
it should be
```
deletion_request = MnoEnterprise::DeletionRequest.new(reason: params[:reason])
deletion_request.relationships.deletable = Organization.find_one(params[:id])
deletion_request.save
```

Reply:
`deletion_request.relationships.deletable = Organization.find_one(params[:id])` does not work as it sets `delatable_type` to `"organizations"` instead of `"Organization"`

Reply:
This is odd, as thatthis is what we did for SubscriptionController and it worked.
relationships.organization = MnoEnterprise::Organization.new(id: parent_organization.id)
I would think that we need to update mno_hub to support having deletable_type to "organizations"

Reply:
The deletable relation is indeed missing on the DeletionRequest. I will update it.